lib/rules/complexity.js

```diff
@@ -133,9 +133,7 @@ module.exports = {
          * @private
          */
         function increaseLogicalComplexity(node) {
-
-            // Avoiding &&
-            if (node.operator === "||") {
+            if (node.operator === "&&" || node.operator === "||") {
```

Comment:
Could this `if` statement be removed?

Reply:
In fact, I can remove the whole function!